templates/default/elasticsearch-env.sh.erb

```diff
@@ -0,0 +1,26 @@
+# JVM Configuration for ElasticSearch
```

Comment:
I'm wondering if we need this file at all. What if we just rely on proper environment variables like `ES_HEAP_SIZE` being exported, and the init scripts either using them or setting them when they are not exported?